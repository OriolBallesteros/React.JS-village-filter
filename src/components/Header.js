import React from 'react';
import { NavLink } from 'react-router-dom';
import '../styles/Header.scss';

const Header = () => (
    <header className='Header'>
        <h1>Meet people in Brastlewark</h1>
        <NavLink to="/" className='NavLink' activeClassName="is-active" exact={true}>Go to filter</NavLink>
        <NavLink to="/searched" className='NavLink' activeClassName="is-active">Listed</NavLink>
    </header>
);

export default Header;