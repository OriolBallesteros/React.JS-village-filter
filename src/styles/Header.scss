.Header {
    text-align: center;
    width: 99%;
    border: 2px solid black;
    border-bottom-left-radius: 20px;
    border-bottom-right-radius: 20px; 
    padding: 0;
    padding-bottom: 20px;
    position: fixed;
    top: 0;
    z-index: 1;
    background: #009999;

    h1{
        font-family: 'Srisakdi', cursive;
    }
}

.NavLink{
    margin: 10px;
}

.is-active {
    font-weight: bold;
}