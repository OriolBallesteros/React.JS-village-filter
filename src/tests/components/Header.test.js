import React from 'react';
import {shallow} from 'enzyme';
import Header from '../../components/Header';

it('should render App component correctly', ()=>{
    const wrapper = shallow(<Header/>);

    expect(wrapper).toMatchSnapshot();
});