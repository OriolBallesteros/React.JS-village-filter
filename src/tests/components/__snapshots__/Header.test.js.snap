// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component correctly 1`] = `
<header
  className="Header"
>
  <h1>
    Meet people in Brastlewark
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    className="NavLink"
    exact={true}
    to="/"
  >
    Go to filter
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    className="NavLink"
    to="/searched"
  >
    Listed
  </NavLink>
</header>
`;
